frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.test.tsx

```diff
@@ -22,8 +24,37 @@ const lCodes = {
   ] as ILookupCode[],
 };
 
+const defaultParcel: IParcel = {
```

Comment:
you may be able to just reuse a parcel from mockDetails.

Reply:
FYI the benefit of that is if we update the IParcel object we only need to fix the mocks in one place instead of... many.

Reply:
right you are!